.github/workflows/docker-test.yml

```diff
@@ -0,0 +1,15 @@
+name: Nightly Docker test
```

Comment:
maybe in the future we could figure out api keys in github to run periodic integration tests!

Reply:
resolving this as it's not an actual request, just a comment